config/drupal-8/drupal-8.8-deprecations.php

```diff
@@ -33,4 +34,6 @@
     $rectorConfig->rule(FileScanDirectoryRector::class);
     $rectorConfig->rule(FileDirectoryTempRector::class);
     $rectorConfig->rule(FileUriTargetRector::class);
+
+    $rectorConfig->rule(FunctionalTestDefaultThemePropertyRector::class);
```

Comment:
Well. This should have failed in the fixtures: `fixtures/d8/rector_examples/test/src/Functional/BrowserTestBaseGetMock.php`

```
<?php

namespace Drupal\Tests\rector_examples\Functional;

use Drupal\Core\Entity\EntityTypeManagerInterface;
use Drupal\Tests\BrowserTestBase;

class BrowserTestBaseGetMock extends BrowserTestBase {

  /**
   * A simple example using the class property.
   */
  public function simple_example() {
      $this->entityTypeManager = $this->getMock(EntityTypeManagerInterface::class);
  }

  /**
   * A simple example using a string directly.
   */
  public function class_name_as_string() {
    $this->entityTypeManager = $this->getMock('Drupal\Core\Entity\EntityTypeManagerInterface');
  }
}
```